setup.py

```diff
@@ -7,7 +7,7 @@
 
 # Always prefer setuptools over distutils
 from pwem import EM_PROGRAM_ENTRY_POINT, CHIMERA_ENTRY_POINT, \
```

Comment:
This import with \ is not pep8 compliant.
I think we can just import pwem and use pwem.XXX as prefix of the constants.

Reply:
I'll fix in devel-fixes